protocol/xmrmaker/errors.go

```diff
@@ -21,6 +21,8 @@ var (
 	errLockTxReverted              = errors.New("other party failed to lock ETH asset (transaction reverted)")
 	errInvalidETHLockedTransaction = errors.New("eth locked tx was not to correct contract address")
 	errRelayerCommissionTooHigh    = errors.New("relayer commission must be less than 0.1 (10%)")
+	errInvalidT0                   = errors.New("invalid t0 value set by counterparty")
+	errInvalidT1                   = errors.New("invalid t1 value set by counterparty")
```

Comment:
Should these messages be something like:

`ETH asset locked too far in the past`
`swap timeout set to unexpected value`

To get the clearest error messages, we should probably check the T1-T0 delta (the swap timeout) before checking the T0 value.

Reply:
`ETH asset locked too far in the past` isn't necessarily true, the counterparty could have just put an invalid timeout. should I change the `errInvalidT1` message to say `delta between t0 and t1 is invalid`?

Reply:
Alice doesn't set the T1-T0 delta directly.  We are using the delta as a proxy to determine if she passed the correct `_timeoutDuration` when calling `newSwap`. That's why I thought a message about the "timeout" or "timeout duration" being incorrect would be more helpful to users.

> ETH asset locked too far in the past isn't necessarily true, the counterparty could have just put an invalid timeout.

That's why I suggested that we flip the order of the checks and eliminate the possibility that the delta was set incorrectly first before performing the 2nd check.